src/components/card/CardSidebar.vue

```diff
@@ -133,21 +138,30 @@ export default {
 		}
 	},
 	watch: {
-		'currentCard': {
-			immediate: true,
-			handler() {
-				this.copiedCard = JSON.parse(JSON.stringify(this.currentCard))
-				this.allLabels = this.currentCard.labels
-				this.assignedUsers = this.currentCard.assignedUsers.map((item) => item.participant)
-				this.desc = this.currentCard.description
-			}
+		currentCard() {
```

Comment:
Any reason you removed the immediate property of the watcher? Iirc that was needed to properly initialize the data on initial load.

Reply:
readded